scripts/scripts.js

```diff
@@ -12,24 +12,114 @@
 
 import { setLibs } from './utils.js';
 
-// Add project-wide style path here.
-const STYLES = '';
-
-// Use '/libs' if your live site maps '/libs' to milo's origin.
-const LIBS = 'https://milo.adobe.com/libs';
-
+const LIBS = '/libs';
+const STYLES = '/styles/styles.css';
 // Add any config options.
 const CONFIG = {
-  // codeRoot: '',
-  // contentRoot: '',
-  // imsClientId: 'college',
-  // geoRouting: 'off',
-  // fallbackRouting: 'off',
+  imsClientId: 'bacom-blog',
+  local: {
+    pdfViewerClientId: '3b685312b5784de6943647df19f1f492',
+    pdfViewerReportSuite: 'adbadobedxqa',
+  },
+  stage: {
+    edgeConfigId: '7d1ba912-10b6-4384-a8ff-4bfb1178e869',
+    pdfViewerClientId: '3b685312b5784de6943647df19f1f492',
+    pdfViewerReportSuite: 'adbadobedxqa',
+  },
+  live: {
+    pdfViewerClientId: '23bd4fff42fc4b4da38b3d89492a0abc',
+    pdfViewerReportSuite: 'adbadobedxqa',
+  },
+  prod: {
+    edgeConfigId: '65acfd54-d9fe-405c-ba04-8342d6782ab0',
+    pdfViewerClientId: '4520c0edfbf147158758d71d18765fec',
+    pdfViewerReportSuite: 'adbadobenonacdcprod,adbadobedxprod,adbadobeprototype',
+  },
```

Comment:
Should check on these configs-maybe make a ticket for it? The pdf client ids are specific to the bacom urls so we'll need to generate new ones for this site. Not sure if the report suites are the same or not. We'd need to ask Malik. 

Not sure about the other configs

Reply:
Checked with Dave, he said everything should be the same, there is a comment on the ticket. I'll remove PDF viewer for now, and we can add it if needed.